src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -366,3 +366,22 @@ def drexel_fast_zonal(geojson, key):
     result = {int(k): v for k, v in res.json()[key].items()}
 
     return result
+
+
+def huc12s_for_huc(huc):
```

Comment:
Initially I named this `get_huc12s`, but that is already defined [elsewhere](https://github.com/WikiWatershed/model-my-watershed/blob/40888b94615e9d2faf869ead097ce971d2e70e1b/src/mmw/apps/modeling/calcs.py#L243).